internal/zinc-ivy-integration/src/main/scala/sbt/internal/inc/ZincComponentCompiler.scala

```diff
@@ -336,7 +335,12 @@ object ZincIvyActions {
     val defaultExcluded = Set("doc")
     val finalExcluded = if (noSource) defaultExcluded + "src" else defaultExcluded
     val artifactFilter = ArtifactTypeFilter.forbid(finalExcluded)
-    UpdateConfiguration(Some(retrieve), missingOk = false, logLevel, artifactFilter)
+    UpdateConfiguration(retrieve = Some(retrieve),
+                        missingOk = false,
+                        logging = logLevel,
+                        artifactFilter = artifactFilter,
+                        offline = false,
```

Comment:
Ideally, we would like these to be configurable from the outside so that if a user enables the offline mode in sbt, we can force it here too. Can you please open a ticket for that?